fdbclient/BlobStore.actor.cpp

```diff
@@ -135,126 +143,131 @@ std::string BlobStoreEndpoint::BlobKnobs::getURLParameters() const {
 	_CHECK_PARAM(max_recv_bytes_per_second, rbps);
-	#undef _CHECK_PARAM
+#undef _CHECK_PARAM
 	return r;
 }
 
-Reference<BlobStoreEndpoint> BlobStoreEndpoint::fromString(std::string const &url, std::string *resourceFromURL, std::string *error, ParametersT *ignored_parameters) {
-	if(resourceFromURL)
-		resourceFromURL->clear();
+Reference<BlobStoreEndpoint> BlobStoreEndpoint::fromString(std::string const& url, std::string* resourceFromURL,
+                                                           std::string* error, ParametersT* ignored_parameters) {
+	if (resourceFromURL) resourceFromURL->clear();
 
 	try {
 		StringRef t(url);
 		StringRef prefix = t.eat("://");
-		if(prefix != LiteralStringRef("blobstore"))
+		if (prefix != LiteralStringRef("blobstore"))
 			throw format("Invalid blobstore URL prefix '%s'", prefix.toString().c_str());
 		StringRef cred = t.eat("@");
 		uint8_t foundSeparator = 0;
 		StringRef hostPort = t.eatAny("/?", &foundSeparator);
 		StringRef resource;
-		if(foundSeparator == '/') {
-			 resource = t.eat("?");
+		if (foundSeparator == '/') {
+			resource = t.eat("?");
 		}
 
 		// hostPort is at least a host or IP address, optionally followed by :portNumber or :serviceName
 		StringRef h(hostPort);
 		StringRef host = h.eat(":");
-		if(host.size() == 0)
-			throw std::string("host cannot be empty");
+		if (host.size() == 0) throw std::string("host cannot be empty");
 
 		StringRef service = h.eat();
 
 		BlobKnobs knobs;
 		HTTP::Headers extraHeaders;
-		while(1) {
+		while (1) {
 			StringRef name = t.eat("=");
-			if(name.size() == 0)
-				break;
+			if (name.size() == 0) break;
 			StringRef value = t.eat("&");
 
 			// Special case for header
-			if(name == LiteralStringRef("header")) {
+			if (name == LiteralStringRef("header")) {
 				StringRef originalValue = value;
 				StringRef headerFieldName = value.eat(":");
 				StringRef headerFieldValue = value;
-				if(headerFieldName.size() == 0 || headerFieldValue.size() == 0) {
-					throw format("'%s' is not a valid value for '%s' parameter.  Format is <FieldName>:<FieldValue> where strings are not empty.", originalValue.toString().c_str(), name.toString().c_str());
+				if (headerFieldName.size() == 0 || headerFieldValue.size() == 0) {
+					throw format("'%s' is not a valid value for '%s' parameter.  Format is <FieldName>:<FieldValue> "
+					             "where strings are not empty.",
+					             originalValue.toString().c_str(), name.toString().c_str());
 				}
-				std::string &fieldValue = extraHeaders[headerFieldName.toString()];
-				// RFC 2616 section 4.2 says header field names can repeat but only if it is valid to concatenate their values with comma separation
-				if(!fieldValue.empty()) {
+				std::string& fieldValue = extraHeaders[headerFieldName.toString()];
+				// RFC 2616 section 4.2 says header field names can repeat but only if it is valid to concatenate their
+				// values with comma separation
+				if (!fieldValue.empty()) {
```

Comment:
We need to define rule for brace wrapping in our `.clang-format`. Some places have single line control statements brace wrapped, and others don't. Should we enforce line breaks after braces, or wrap the block to single line. 

IMO its just easier to have braces everywhere and with no wrapping.